install_config/aggregate_logging.adoc

```diff
@@ -474,6 +495,10 @@ created, and ephemeral volumes are used instead. If you set this parameter, the
 If the `openshift_logging_es_pvc_dynamic` parameter has been set to `false`, you must set a value for this parameter.
 Read the description of `openshift_logging_es_pvc_prefix` for more information.
 
+|`openshift_logging_elasticsearch_image`
+|The image version for Elasticsearch.
+For example: `openshift_logging_elasticsearch_image=<registry_url>:5000/openshift3/logging-elasticsearch:v3.X.X`
```

Comment:
Shall we use offical images as an example? registry.redhat.io/openshift3/ose-logging-elasticsearch5:v3.11